cmake/modules/extensions.cmake

```diff
@@ -1559,6 +1559,66 @@ function(zephyr_syscall_header_ifdef feature_toggle)
   endif()
 endfunction()
 
+# Verify blobs fetched using west. If the sha256 checksum isn't valid, a warning/
+# fatal error message is printed (depends on REQUIRED flag).
+# Paths are relative to the current module's zephyr/blobs directory.
```

Comment:
Not good.

Please let code accept absolute paths, and when a path is relative, then it should be relative to `CMAKE_CURRENT_SOURCE_DIR` which is most common in CMake.
Will also allow the function to be called outside of a module, for example like:
```
zephyr_blobs_verify(FILES ${ZEPHYR_HAL_INFENION_MODULE_DIR}/zephyr/blobs/img/cat1cm0p/COMPONENT_CM0P_SLEEP/psoc6_01_cm0p_sleep.bin)
```

Or when inside a module's CMakeLists.txt file, developer may do like:
```
zephyr_blobs_verify(FILES
    ${bless_blobs_dir}/COMPONENT_BLESS_CONTROLLER/cy_ble_stack_controller.a
  	${bless_blobs_dir}/COMPONENT_BLESS_CONTROLLER/cy_ble_stack_manager.a
)
```
in a place like here:
https://github.com/zephyrproject-rtos/zephyr/blob/4403d8f4c622fe858a57aeea4b555256102fa890/modules/hal_infineon/bless/CMakeLists.txt#L20-L23